server/game/cards/02_SHD/units/TheClientDictatedByDiscretion.ts

```diff
@@ -0,0 +1,36 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { CardType, KeywordName, WildcardCardType } from '../../../core/Constants';
+
+export default class TheClientDictatedByDiscretion extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '1090660242',
+            internalName: 'the-client#dictated-by-discretion'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addActionAbility({
+            title: 'Choose a unit. For this phase, it gains: "Bounty — Heal 5 damage from a base."',
```

Comment:
nit: can remove the "choose a unit" part since the player will already be looking at a target selection prompt. Might need to reword it grammatically to make sense though, maybe `For this phase, target unit gains ...`